runtime/compiler/runtime/RelocationRecord.cpp

```diff
@@ -1799,6 +1809,33 @@ TR_RelocationRecordThunks::relocateAndRegisterThunk(TR_RelocationRuntime *reloRu
    return 0;
    }
 
+// TR_J2IVirtualThunkPointer Relocation
+char *
+TR_RelocationRecordJ2IVirtualThunkPointer::name()
+   {
+   return "TR_J2IVirtualThunkPointer";
+   }
+
+int32_t
+TR_RelocationRecordJ2IVirtualThunkPointer::bytesInHeaderAndPayload()
+   {
+   return sizeof(TR_RelocationRecordJ2IVirtualThunkPointerBinaryTemplate);
+   }
+
+void
+TR_RelocationRecordJ2IVirtualThunkPointer::relocateJ2IVirtualThunkPointer(
+   TR_RelocationTarget *reloTarget,
+   uint8_t *reloLocation,
+   void *thunk)
+   {
```

Comment:
I'd stick an assert here that `thunk != NULL` just for completeness

Reply:
Assertion added